mlir/test/rocmlir-driver/auto_e2e/conv2d_f32_wrw_perf_config.mlir

```diff
@@ -0,0 +1,3 @@
+// RUN: rocmlir-gen --operation conv2d_bwd_weight --perf_config 64,64,2,64,64,2,1,1 -t f32 -x2 --fil_layout kcyx --in_layout nchw --out_layout nkhw --batchsize 512 --in_channels 512 --in_h 1 --in_w 1 --out_channels 512 --fil_h 1 --fil_w 1 --dilation_h 1 --dilation_w 1 --conv_stride_h 1 --conv_stride_w 1 --padding_h 0 --padding_w 0 %pv %random_data -x2 | rocmlir-driver -c | mlir-rocm-runner --shared-libs=%linalg_test_lib_dir/libmlir_rocm_runtime%shlibext,%conv_validation_wrapper_library_dir/libconv-validation-wrappers%shlibext,%linalg_test_lib_dir/libmlir_runner_utils%shlibext --entry-point-result=void | FileCheck %s --check-prefix=CHECK_F32_WRW_PERF_CFG
```

Comment:
@manupak Do you expect this test to be xdlops-only? 
- If so, it needs to be put in a directory, like the deleted PaddingXDLOPS, with a lit.local.cfg to make sure this test runs only when xdlops is enabled, i.e. Jenkins picked gfx908 or gfx90a.
- If this test needs to be checked for both xdlops and non-xdlops, `-x2` needs to be replaced by `%xdlops`.

Just want to check which is the case. I can fix them in my current PR.

Reply:
Yeah, good catch @zhanglx13. Making this %xdlops or moving it to a relevant directory needs to happen today. I'm willing to approve a hotfix PR and to cheat it through CI so we don't have broken nightlies/weeklies

Reply:
Sure.
Please correct me if I am wrong, since I am not familiar with `perf_config`. From the number of values provided to `perf_config`, it seems that this is a xdlops-only test, is that true @krzysz00?

Reply:
Yep, it is

Reply:
My bad.. good catch! Thanks guys!